apps/src/templates/projects/GallerySwitcher.jsx

```diff
@@ -50,31 +48,29 @@ const styles = {
 
 class GallerySwitcher extends Component {
   static propTypes = {
-    initialGallery: PropTypes.oneOf(Object.keys(Galleries)).isRequired,
+    pageLocation: PropTypes.string.isRequired,
```

Comment:
the name `pageLocation` seems misleading since it suggests a URL path like `/projects` or `/projects/public`. What do you think about calling this `selectedGallery`, or anything else without "location" in it? If you do change it, the string should probably be changed everywhere it appears in this PR.